apps/src/netsim/NetSimDnsTable.html.ejs

```diff
@@ -0,0 +1,44 @@
+<%
```

Comment:
Started trying to do this, and realized that haml is part of our ruby pipeline, but apparently not part of our apps pipeline.